demo/pb-search-select.html

```diff
@@ -0,0 +1,113 @@
+<html>
```

Comment:
Examples should be informative for the user. This example should rather be a test case.